package.json

```diff
@@ -2,31 +2,29 @@
   "name": "bobaserver",
   "version": "1.0.0",
   "description": "Server for BobaBoard",
-  "homepage": "https://docs.bobaboard.com/",
+  "homepage": "https://github.com/essential-randomness/bobaserver#readme",
```

Comment:
revert these changes?

Reply:
I'm so confused as to how this happened? I pulled from main before I started working?